webapp/src/index.tsx

```diff
@@ -25,6 +26,7 @@ export default class Plugin {
 
         const channelNoteUpateEvent = `custom_${manifest.id}_channel_note_update`;
         registry.registerWebSocketEventHandler(channelNoteUpateEvent, handleWebsocketNoteUpdate(store.getState, store.dispatch));
+        registry.registerWebSocketEventHandler(WebsocketEvents.CHANNEL_VIEWED, handleWebsocketChannelViewed(store.getState, store.dispatch))
```

Comment:
@agarciamontoro This will potentially have issues if you're using more than one client right? If you browse to another channel in one client, won't the note update for all clients?

Maybe the RHS component needs to handle the state change instead of websockets?

Reply:
@mickmister, thanks for checking!

> If you browse to another channel in one client, won't the note update for all clients?

Mmm, that's true. But that will cause a potential re-fecth that may not me needed, right? It will not create any invalid state if I'm not mistaken. Or am I missing something?

> Maybe the RHS component needs to handle the state change instead of websockets?

We need something that's mounted all the time for the old logic in the channel header icon coloring to keep working. The RHS component is unmounted when the RHS is closed, right?

Reply:
Alternatively, we could relocate the hook-based sync to a root component, that unconditionally renders `null`

Reply:
Ooooh, I understand now the problem. Yes, it will overwrite with the notes from your other client, probably. I'll use the root component solution.